tests/00-setup

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sudo add-apt-repository ppa:juju/stable -y
+sudo apt-get update
+sudo apt-get install amulet python-requests -y
```

Comment:
Are these two test files mainly boilerplate? They are inconsistent.

10-deploy has python3 on the shebang. 00-setup installs python-requests not python3-requests, which I just verified is required for trusty.